examples/agent_motion_prediction/agent_motion_prediction.ipynb

```diff
@@ -191,7 +193,23 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "# Evaluation"
+    "# Evaluation\n",
+    "\n",
+    "Evaluation follows a slightly different protocol from training. When working with time series, we must be absolutely sure to avoid leaking the future in the data.\n",
+    "\n",
+    "If we followed the same protocol of training, one could just read ahead in the `.zarr` and forge a perfect solution at run-time, even for a private test set.\n",
+    "\n",
+    "As such, **the private test set for the competition has been \"chopped\" using the `chop_dataset` function**.\n",
+    "\n",
+    "The result is that each scene has been reduced to only 100 frames, and only valid agents in the 100th frame will be used to compute the metrics. Because following frames in the scene have been chopped off, we can't just look ahead to get the future of those agents.\n",
```

Comment:
Is this really how it's done, only valid agents in the 100th frame - not valid agents in any of the first 100 frames?

Reply:
nope, otherwise you leak the future when you include the following frames